src/reference-manual/expressions.Rmd

```diff
@@ -1231,7 +1235,7 @@ vector[7] beta;
 array[10] int x_i;
 int index;
 // ...
-vector[2] y_hat[T] = ode_rk45(foo, y0, t0, ts, theta, 
```

Comment:
We really need auto formatting to get rid of all this line-final space!

Reply:
Emacs caught this for me automatically, I'm not sure if there is a way to do it for all PRs though

Reply:
Yeah, I have emacs instrumented to remove final space, but I don't know if I have it turned on for .Rmd files, so I may be the offending party for a lot of those.